examples/gallery/lines/connection_lines.py

```diff
@@ -0,0 +1,55 @@
+"""
+Connection lines
+================
+
+The ``connection`` parameter of the :meth:`pygmt.Figure.plot` method allows to plot
+connection lines between a set of data points. Width, color, and style of the lines
+can be adjusted via the ``pen`` parameter. The data points must be plotted separately
+using the ``style`` parameter, with adjustments for the symbol fill and outline via
+the ``fill`` and ``pen`` parameters, respectively.
+"""
+
+# %%
+import pygmt
+
+# Set up same sample data
+x_cords = [2.2, 3.3, -3.1, -3.7, -0.1]
+y_cords = [1.8, -1.2, -0.9, -4.5, 4.5]
```

Comment:
Maybe just rename `x_cords`/`y_cords` to `x`/`y`?

Reply:
Renamed in commit https://github.com/GenericMappingTools/pygmt/pull/2999/commits/bc7b021fef60e84d851e815dbe4f82a0ef8ab4e3.